.github/workflows/e2e.yml

```
@@ -162,10 +162,51 @@ jobs:
       - name: Run e2e test
         run: timeout 2400 bats "e2e/$E2E_TEST"
 
+  ui_test:
+    runs-on: ${{ matrix.os }}
+    needs: [build_dfx]
+    strategy:
+      matrix:
+        os: [macos-12, ubuntu-20.04, ubuntu-22.04]
+        rust: ['1.65.0']
```

we recently bumped rust version in our repo, which is now causing your CI to fail, you'll need to change this 
```suggestion
        rust: ['1.77.1']
```
sorry for the trouble 🥲

---

and if we merge [this PR](https://github.com/dfinity/sdk/pull/3286) it'll be wrong again, too...

---

why? I don't think it will